src/features/rewards/welcomePage/style.ts

```diff
@@ -102,6 +115,10 @@ export const StyledSubTitle = styled(Heading)`
   text-align: center;
   line-height: 28px;
   margin: 18px 0 7px;
+
+  @media (max-width: 360px) {
+    font-size: 22px;
```

Comment:
Same re: Heading. Otherwise is there much point in using the Heading component if we're setting the font-size here?